crates/network-scanner-net/src/lib.rs

```diff
@@ -1 +1,11 @@
-pub mod tokio_raw_socket;
+pub mod async_io;
+pub mod async_raw_socket;
```

Comment:
naming: I would suggest this:
```suggestion
pub mod runtime;
pub mod socket;
```